### ISSUE TITLE
External Networking to access the Local Area Network

### ISSUE THREAD

**Issue by lmntjames** [vladikoff/chromeos-apk#39]
Just bringing this up in hopes that someone with more knowledge on chrome apps can assist.  When attempting to package networking tools such as <a href="https://play.google.com/store/apps/details?id=com.overlook.android.fing">Fing</a> or <a href="https://play.google.com/store/apps/details?id=com.farproc.wifi.analyzer">WiFi Analyzer</a> you are able to see that the apps are isolated from the LAN; Wi-Fi analyzer, for example, displays 127.0.0.1  I messed with permissions in manifest.json but was unsuccessful in getting these apps to communicate with other devices on the LAN.  Maybe this is something that needs to be tweaked in ARChron or it may be beyond the scope of this project.  But given the list of attempted apps that do not work I think if this was sorted out we would have even more successes.

**Comment by pavangupta** [comment]
Yep, definitely seems like a thing.  Experiencing essentially this problem with the Xbox Smartglass app (both beta and normal).  It ends up unable to find the xbox on the local network (while clearly accessing the outside.)  Image below shows the normal, not particularly descriptive failure, but if you look closely, you'll see a bunch of content definitely loaded from the internet.
![screenshot 2015-02-25 01 09 30](https://cloud.githubusercontent.com/assets/1335171/6365801/0798814c-bc8b-11e4-8752-fa60e4b1859d.png)

And just for the sake of being complete, entering the IP address directly doesn't prod it into working, unfortunately.

![image](https://cloud.githubusercontent.com/assets/1335171/6365826/52666fb8-bc8b-11e4-9404-283f0e20f49d.png)
